tests/script_doctrine_dbal.sh

```diff
@@ -1,8 +1,5 @@
 #!/bin/bash
 
-composer require jackalope/jackalope-doctrine-dbal:"~1.0" --no-update
-composer update $COMPOSER_FLAGS --prefer-source --no-interaction;
```

Comment:
the idea with these bash scripts was that you can also build the project locally. but i am ok to change this - it is only a one-time step (and tricky to test locally anyways)

Reply:
I would say it is better to have a makefile to do that.